### ISSUE TITLE
Translation support for Brazilian Portuguese (pt_BR)

### ISSUE THREAD

**Issue by wtuemura** [Marat-Tanalin/bsnes-mt#7]
Here is the 100% translation for Brazilian Portuguese, Brazil it's incompatible with ISO 639-1, we use [RFC-5646](https://tools.ietf.org/html/rfc5646) or [language tag](https://en.wikipedia.org/wiki/Language_localisation#Language_tags_and_codes) since pt is Portugal Portuguese.

For Brazilian Portuguese it can be pt_BR or pt-BR.
Thanks.

[pt_BR.zip](https://github.com/Marat-Tanalin/bsnes-mt/files/5226673/pt_BR.zip)

**Comment by Marat-Tanalin** [comment]
Thanks. Will add support for subtags then.

**Comment by Marat-Tanalin** [comment]
Implemented support for language subtags and custom locales, and added your Brazilian-Portuguese translation as `pt-br.txt`.

I applied a minor quote-related fix to the original translation file: code quotes (`""`) was contained instead of typography quotes (`“”`) in a string, that broke parsing translation-file where code quotes are delimiters that translated strings are wrapped&nbsp;in.

Thank you for your contribution, Wellington.
